javascript/javascript2/week3/fastestpresser.js

```diff
@@ -0,0 +1,82 @@
+"use strict"
+const button_tag = document.getElementById("start_button");
+const input_tag = document.getElementById("time_input")
+const gameInfo = document.getElementById("game_info");
+
+let countS = 0;
+let countL = 0;
+function countOfKeys(e) {
+    console.log(e)
+    const count1_tag = document.getElementById("count_S");
+    const count2_tag = document.getElementById("count_L");
+    if (e.code === "KeyS") {
+        countS++;
+        count1_tag.innerHTML = countS;
+    } else if (e.code === "KeyL") {
+        countL++
+        count2_tag.innerHTML = countL;
+    } else {
+        gameInfo.innerHTML = "please enter right key"
+    }
+}
+
+function gameend() {
+    document.removeEventListener("keypress", countOfKeys)
+    if (countS > countL) {
+        gameInfo.innerHTML = "WINNER : S-key user";
+        var confettiSettings = { target: 'S-canvas' };
```

Comment:
Here, you could have used const or let instead of var. An other thing that you could do, is that since you are using the same code for when L wins and when S wins you could have the confetti code in a function and call it from the if statements.